magento/magento2
# 8862: Can't emptying special price value by magento 2 api.

### Description
Fix for: Can't emptying special price value by magento 2 api.

### Fixed Issues (if relevant)

1. magento/magento2#8862: Can't emptying values by magento 2 api

### Manual testing scenarios
1. Login to admin.
2. Create simple product with special price.
3. Make API request. GET rest/all/V1/products/{product_sku}.
4. Modify response, set special price, special_from_date, spacial_to_date values to empty string. Wrap response body with { "product": your_response_body_as_json }
5. Make API request, PUT rest/all/V1/products/{product_sku} with modified response as body.
6. Open product on admin product page. Check it hasn't special price anymore.

### Contribution checklist
